lib/fork.js

```diff
@@ -2,7 +2,12 @@ const childProcess = require('child_process')
     , childModule  = require.resolve('./child/index')
 
 function fork (forkModule) {
-  var child = childProcess.fork(childModule, {
+  // suppress --debug flags while preserving others (like --harmony)
+  var filteredArgs = process.execArgv.filter(function (v) {
+    return v.indexOf('--debug') === -1
+  })
+
+  var child = childProcess.fork(childModule, {execArgv: filteredArgs}, {
```

Comment:
see the other files for how multiple `var` statements should be avoided, comma-first, indent the previous `filter` block to match.

spaces after `{` and before `}` here too please.